experiment.py

```diff
@@ -1,58 +1,48 @@
 import asyncio
+import logging
 import os
 from pprint import pprint
 
-import httpx
-from awesomeversion import AwesomeVersion
-
 from pyenphase.envoy import Envoy
 
+logging.basicConfig(level=logging.DEBUG)
+
 
 async def main() -> None:
     envoy = Envoy("envoy.local")
```

Comment:
```suggestion
    envoy = Envoy(os.environ.get("ENVOY_HOST", "envoy.local"))
```
Maybe easier to test multiple instances with